src/c/core/serialization/xrce_protocol.c

```diff
@@ -282,16 +283,16 @@ bool deserialize_TransportLocatorSeq(MicroBuffer* buffer, TransportLocatorSeq* o
 bool serialize_Property(MicroBuffer* buffer, const Property* input)
 {
     bool ret = true;
-    ret &= serialize_String_t(buffer, &input->name);
-    ret &= serialize_String_t(buffer, &input->value);
+    ret &= serialize_sequence_char(buffer, input->name, (uint32_t)(strlen(input->name) + 1));
+    ret &= serialize_sequence_char(buffer, input->value, (uint32_t)(strlen(input->value) + 1));
     return ret;
 }
 
 bool deserialize_Property(MicroBuffer* buffer, Property* output)
 {
     bool ret = true;
-    ret &= deserialize_String_t(buffer, &output->name);
-    ret &= deserialize_String_t(buffer, &output->value);
+    ret &= deserialize_sequence_char(buffer, output->name, NULL);
+    ret &= deserialize_sequence_char(buffer, output->value, NULL);
```

Comment:
`NULL` value generates a *core exception* because `deserialize_sequence_char` writes in that address. Will be the function `deserialize_Property` called by someone or is deprecated?

Reply:
It was fixed. It should be modified with the new Micro CDR API.